less/core/global.less

```diff
@@ -6,3 +6,25 @@ body {
   display: inline-block;
   vertical-align: middle;
 }
+
+// Quote related styling
+// override browser defined margins with rem unit
+blockquote,
+figure {
+  margin: @item-margin * 2;
+}
+
+// styled quote with attribution
+.figure-quote blockquote,
+.figure-quote q {
```

Comment:
I'm not sure about including the <q> here as that tag is for inline quotes - brief quotations that do not require paragraph breaks. So I'd imagine the only time it'd be used the text would be the same as the body text around it (not @body-size * 1.5).

Reply:
You're correct, the blockquote and q elements don't need to be specifically called out in the css here. 

Rather confusingly, you can have a myriad of these html tags intertwined with each other in various ways and it can all be used correctly. 

The use of the blockquote and q tags aren't exclusive to the figure element, they can be used outside of this tag with only minor styling amends attached to the blockquote. The introduction of the figure element here will gives us benefits in the form of: 

1. Allowing for semantically correct quotation, be it via blockquote or q, with an author attribution
2. Allowing for the attachment of a custom class for unique styling

Reply:
Yep, I was just referring the styling of the < q > as both could be used in the same document. 

Agree, if using < figcaption > , < figure > should wrap < blockquote > 

As we've lost the < q > tag in the css, perhaps we should name the variable blockquote-XXX rather than just quote-XXX as its just referring to the actual < blockquote > tag. But thats just a minor thing perhaps. 

I'd also say if we are differentiating between a block quote and a true pull quote, actual Pull quotes should be marked up with a < aside > tag around the lot - so maybe sensible to add some css or a selector for that too similar to your 'figure-quote'.

Reply:
I agree that the q tag should inherit the styling it is surrounded by as there could be multiple on a page. The current set up should allow for that. A q tag used outside of the `figure-quote` class will inherit the body styling - a q tag used within will inherit styles from the `quote-body` mixin. 

The only blockquote style within the `figure-quote` class is to remove the double margin indent that comes from using a blockquote tag within a figure tag. I've left the styling open so both blockquote and q can be - it's not limited to purely blockquote.

Agree that something should exist to support a semantically correct implementation of a true pull quote with some refined styling.

Reply:
Yep, I saw all that - all good. Just thought maybe the less variable names could be specific now. But not precious about that.